updating/understanding-openshift-upgrade-duration.adoc

```diff
@@ -0,0 +1,96 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-upgrade-duration"]
+= Understanding {product-title} upgrade duration
+include::_attributes/common-attributes.adoc[]
+:context: understanding-openshift-upgrade-duration
+
+toc::[]
+
+{product-title} upgrade duration varies based on the deployment topology. This section describes the approximate time duration of each component that follows through during the cluster update. It also explains the factors that affect the upgrade duration and the applicable {product-title} parameters.
+
+From {product-title} 4 onward, {op-system-first} is the only supported operating system (OS) for cluster upgrades. For a cluster with xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[{op-system-base} worker nodes] must be upgraded separately.
+
+== Prerequisites
+* You are familiar with xref:../architecture/architecture.adoc#architecture[OpenShift Container Platform architecture] and xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[OpenShift Container Platform updates].
+
+== Cluster upgrade duration
+
+Historical upgrade duration of similar clusters provides you the best estimate for the future cluster upgrades. However, if the historical data is not available, you should consider the time estimate of the following components to find your cluster upgrade duration:
+
+* Cluster Version Operator (CVO)
+* Machine Config Operator (MCO)
+* Pod Disruption Budget (PDB)
+* Cluster node health
+* Number of nodes in the cluster
+* {op-system-base-full} worker nodes
+
+=== Cluster Version Operator target upgrade payload deployment
+The Cluster Version Operator (CVO) retrieves the target upgrade release image and applies to the cluster. This process might take somewhere between 45 to 120 minutes.
```

Comment:
45min-120min seems a big range, do you think we need clarify for what kind of cluster may need such a duration for cvo target payload deployment? Something like M master nodes with default `maxUnavailable`.

Reply:
@jiajliu FYI, It's tracked in [OSDOCS-2429](https://issues.redhat.com/browse/OSDOCS-2429)

Reply:
ack, thanks